File: C++ for Everyone, 2nd Ed, Horstmann/Chapter2/P2-3.cpp
//Write a program that reads a number and displays the square, cube, and fourth
//power.Use the  pow function only for the fourth power.

#include<iostream>
#include<cmath>

using namespace std;

int main()
{
    double n,
        nSquare,
        nCube,
        nQuad;

    cout << "Enter a number: ";
    cin >> n;

    nSquare = n*n;
    nCube = pow(n, 3);
    nQuad = pow(n, 4;
    cout << "The square, cube and quad of "<<n<<" is "<< nSquare << ", " << nCube << ", " << nQuad;

    return 0;
}

File: Python For Everyone, Horstmann/Chapter2/P2.3.py
# Write a program that reads a number and displays the square, cube, and fourth
# power. Use the  ** operator only for the fourth power.


number = float(input("Enter a number: "))

numberSquare = number * number
numberCube = number * number * number
numberFourth = number ** 4

print("User input number: %.2f" % number)
print("Square of number: %.2f" % numberSquare)
print("Cube of number: %.2f" % numberCube)
print("Fourth power of number: %.2f" % numberFourth)